eox_core/api/v1/permissions.py

```diff
@@ -40,7 +40,11 @@ def has_permission(self, request, view):
         To grant access, checks if the requesting user either can call eox-core API or if it's an admin user.
         """
         try:
-            allowed_for_site = request.user.is_staff or request.get_host() in request.auth.client.url
+            if request.user.is_staff:
+                allowed_for_site = request.user.is_staff
+            else:
+                allowed_for_site = request.auth.application.redirect_uri_allowed(request.build_absolute_uri('/')) or \
```

Comment:
This always raises an exception, doesn't it ? when the request is with dop

Reply:
No, when the request is with dop tries with:
`request.get_host() in request.auth.client.url`